--- ../3rdparty/SafeInt/SafeInt.hpp.orig	2019-01-21 04:44:44.723753013 +0000
+++ ../3rdparty/SafeInt/SafeInt.hpp	2019-01-21 04:45:12.687599562 +0000
@@ -5815,6 +5815,7 @@
     const T* Ptr() const SAFEINT_NOTHROW { return &m_int; }
     _CONSTEXPR14 const T& Ref() const SAFEINT_NOTHROW { return m_int; }
 
+#if 0
     // Or if SafeInt< T, E >::Ptr() is inconvenient, use the overload
     // operator &
     // This allows you to do unsafe things!
@@ -5822,6 +5823,7 @@
     // pass a SafeInt into things like ReadFile
     T* operator &() SAFEINT_NOTHROW { return &m_int; }
     const T* operator &() const SAFEINT_NOTHROW { return &m_int; }
+#endif
 
     // Unary operators
     _CONSTEXPR11 bool operator !() const SAFEINT_NOTHROW { return (!m_int) ? true : false; }
